src/ui/HiddenEvents.cpp

```diff
@@ -17,9 +17,19 @@ HiddenEvents::load()
     hiddenEvents.hidden_event_types = std::vector{
       EventType::Reaction,
       EventType::CallCandidates,
+      EventType::CallNegotiate,
       EventType::Unsupported,
     };
 
+    //check if selected answer is from to local user
+    /*
+        * localUser accepts/rejects the call and it is selected by caller - No message
+        * Another User accepts/rejects the call and it is selected by caller - "Call answered/rejected elsewhere"
+    */
+    bool callLocalUser_ = true;
+    if(callLocalUser_)
+        hiddenEvents.hidden_event_types->push_back(EventType::CallSelectAnswer);
```

Comment:
Hm, that might not make much sense :D

I think we should either always show that event or hide it without giving the user an option to unhide it.

Reply:
I actually meant for that variable to hold a bool based on whether the message is sent by the local user on local device or otherwise and not an option the user can manually toggle. I'll remove it as of now.